openpype/hosts/traypublisher/plugins/publish/collect_explicit_colorspace.py

```diff
@@ -26,20 +26,43 @@ class CollectColorspace(pyblish.api.InstancePlugin,
 
     def process(self, instance):
         values = self.get_attr_values_from_data(instance.data)
-        colorspace = values.get("colorspace", None)
-        if colorspace is None:
+        colorspace_value = values.get("colorspace", None)
+        if colorspace_value is None:
             return
 
-        self.log.debug("Explicit colorspace set to: {}".format(colorspace))
+        color_data = colorspace.convert_colorspace_enumerator_item(
+            colorspace_value, self.config_items)
+
+        colorspace_name = self._colorspace_name_by_type(color_data)
+        self.log.debug("Explicit colorspace name: {}".format(colorspace_name))
 
         context = instance.context
+        context.data["colorspaceConfigItems"] = self.config_items
```

Comment:
Why are we storing these in context, I don't see them used in this PR?

Reply:
yeah, this had been commited by mistake.